Examples/SqlExample.py

```diff
@@ -27,21 +27,22 @@
 
 """
 
-import datetime
 import os
 import os.path
 import re
-import time
+# import time
```

Comment:
Why is this commented out rather than removed?

Reply:
It's easy: I simply forgot to remove this.